ReactWindows/ReactNative.Shared/ReactRootView.cs

```diff
@@ -110,27 +128,53 @@ public void StartReactApplication(ReactInstanceManager reactInstanceManager, str
             _jsModuleName = moduleName;
             _initialProps = initialProps;
 
-            var getReactContextTask = default(Task);
-            if (!_reactInstanceManager.HasStartedCreatingInitialContext)
-            {
-                getReactContextTask = _reactInstanceManager.GetOrCreateReactContextAsync(CancellationToken.None);
-            }
-
+            var getReactContextTaskTask =
```

Comment:
>getReactContextTaskTask [](start = 16, length = 23)

Can you just use `Unwrap` on the outer task here to reduce the diff?

Reply:
Learning something every day.

---
In reply to: [165945247](https://github.com/Microsoft/react-native-windows/pull/1615#discussion_r165945247) [](ancestors = 165945247)